client/header/activity-panel/wordpress-notices.js

```diff
@@ -135,6 +135,12 @@ class WordPressNotices extends Component {
 		if ( 'message' === element.id && 'updated' === element.className ) {
 			return false;
 		}
+
+		if ( 'dolly' === element.id ) {
+			element.style.display = 'none';
```

Comment:
I don't think anyone is going to complain if this gets hidden, but we can also fix the layout issues by changing the float.

```suggestion
			element.style.cssFloat = 'none';
```

Reply:
The problem with this is that then the lyrics don't always show up in the same location. In WC appears in one place, WordPress another. So I think it makes the most sense to hide it.